templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,10 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 65**
+
+* `POST /realm/playgrounds`: Renamed `name` field to `playground_name`.
```

Comment:
I'm not sure we want to change the field name in the API over this; `playground_` would need to be a prefix for all the field names for this to be consistent.  I had just been imagining a change to the field name in the form with some rewriting in the JS code before sending a request to the server.

Reply:
Yup, I agree. I pushed back this change to the PR.